src/scanner.rs

```
@@ -2365,7 +2374,9 @@ impl<T: Iterator<Item = char>> Scanner<T> {
             );
             self.roll_one_col_indent();
 
-            self.simple_keys.last_mut().unwrap().possible = false;
```

Removing this unwrap would silence a potential error in the scanner logic.